db/seeds.rb

```diff
@@ -1,2 +1,59 @@
 # This file should contain all the record creation needed to seed the database with its default values.
 # The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
+# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
+ActiveRecord::Base.transaction do
+  AdminUser.where('username': 'admin').first_or_create!(
+    username: 'admin',
+    email: 'admin@domain.tld',
+    password: 'adminadmin',
+    password_confirmation: 'adminadmin',
+    identity_code: '38001085718',
+    country_code: 'EE',
+    roles: ['admin']
+  )
+  # Required for creating registrar
+  Setting.where('var': 'registry_vat_prc').first_or_create(
```

Comment:
Duplicates https://github.com/internetee/registry/blob/master/config/app.yml#L59.

Reply:
I could not figure out how to load defaults variables from app.yml it isn't done automatically